### ISSUE TITLE
[BUG] Qwen1.5-14B回答不会停止，会自动进行对话，或者出现重复回答

### ISSUE THREAD

**Issue by Parasolation** [QwenLM/Qwen#1172]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

采用本库给的cli_demo.py或者web_demo.py启动都会遇到该问题
![image](https://github.com/QwenLM/Qwen/assets/51755826/4823ea08-dba2-4acc-b427-c6c011b16915)
![image](https://github.com/QwenLM/Qwen/assets/51755826/f0363d29-1b08-4e36-b3d5-50bbf3cfc191)

### 期望行为 | Expected Behavior

希望能得到正确回答，如上图只需要回答“你好！有什么我可以帮助你的吗？”以及重复的只回答一遍

### 运行环境 | Environment

```Markdown
- OS:Ubuntu22.04
- Python:3.10
- Transformers:4.39.1
- PyTorch:2.21
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):12.1
```

**Comment by Parasolation** [comment]
还会有这样的提示：
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation.
